docs/docs/guide/usingFreighterBrowser.md

```diff
@@ -0,0 +1,191 @@
+---
+id: usingFreighterBrowser
+title: Using Freighter in the browser
+---
+
+We now have an extension installed on our machine and a library to interact with it. This library will provide you methods to send and receive data from a user's extension in your website or application.
+
+### Importing
+
+First import the library in the `<head>` tag of your page.
+
+```html
+<head>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/stellar-freighter-api/{version}/index.min.js"></script>
```

Comment:
Do users need to enter `version` manually? If so, maybe it's worth adding a note here about it.